src/liteclient.h

```diff
@@ -88,7 +88,7 @@ class LiteClient {
   FRIEND_TEST(AkliteTest, RollbackIfAppsInstallFails);
   FRIEND_TEST(AkliteTest, RollbackIfAppsInstallFailsAndPowerCut);
 
-  void callback(const char* msg, const Uptane::Target& install_target, const std::string& result = "");
+  virtual void callback(const char* msg, const Uptane::Target& install_target, const std::string& result);
```

Comment:
A bit odd that we have to make it "virtual" just for sake of testing. I wonder if it is possible to test it without making this method virtual?

Reply:
It makes testing much harder. The production code would have to be changed even more if we don't make the method virtual and still want to mock it (a template would have to be used). Since the performance penalty is negligible, I believe it is ok to make this concession in production code to favor testing.

Reply:
ok.